system/HTTP/CLIRequest.php

```diff
@@ -272,6 +272,20 @@ public function getGetPost($index = null, $filter = null, $flags = null)
         return $this->returnNullOrEmptyArray($index);
     }
 
+    /**
+     * This is a place holder for calls from cookie_helper get_cookie().
+     *
+     * @param array|string|null $index  Index for item to be fetched from $_COOKIE
+     * @param int|null          $filter A filter name to be applied
+     * @param mixed             $flags
+     *
+     * @return null
+     */
+    public function getCookie($index = null, $filter = null, $flags = null)
+    {
+        return null;
```

Comment:
It may return `array` when `$index` is null.
Please use `$this->returnNullOrEmptyArray($index);`.